include/haywire.h

```diff
@@ -172,7 +172,7 @@ HAYWIRE_EXTERN int hw_init_from_config(char* configuration_filename);
 HAYWIRE_EXTERN int hw_init_with_config(configuration* config);
 HAYWIRE_EXTERN int hw_http_open();
 HAYWIRE_EXTERN void hw_http_add_route(char* route, http_request_callback callback, void* user_data);
-HAYWIRE_EXTERN char* hw_get_header(http_request* request, char* key);
+HAYWIRE_EXTERN hw_string* hw_get_header(http_request* request, hw_string* key);
```

Comment:
We can also add a version that takes a `char*` as key. We would just need to wrap it in a `hw_string` for the lookup.